apps/hub-app/src/pages/App.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styled from 'styled-components';
+import Header from '../components/Header';
+import Profile from '../components/Profile';
+import { crimsonDark } from '@radix-ui/colors';
+
+const Layout = styled.div`
```

Comment:
I wonder if we'll want to generalize some of our Layout components and consider adding them to the design system for reusability?